docker-compose.yml

```diff
@@ -33,7 +33,8 @@ services:
       - ${DOCKER_SHARED_VOLUME_WINDOWS}/var/run/docker.sock:/var/run/docker.sock
 
   ode:
-    image: usdotjpoode/jpo-ode:1.0
+    build: .
```

Comment:
Everything looks good. A minor question whether building the image from source or pull from this dockerhub: https://hub.docker.com/u/usdotjpoode?

Reply:
Building from source ensures it is using all the code changes that have been made within the ODE portion of the project although building from the image is good if it is the latest and the new changes have been published if I am not mistaken.